admin/build-doc

```diff
@@ -20,7 +20,6 @@ if command -v dpkg >/dev/null; then
       graphviz
       ant
       cython
-      librbd-dev
```

Comment:
this partially reverts the change in #9147. @flaf could you help review it? thanks.